### DISCUSSION TITLE
Rails instrumentation for Rails LTS (3+)

### DISCUSSION THREAD

**Discussion by GBoucher42** [open-telemetry/opentelemetry-ruby-contrib#1177] [General]
Hi, looking at the code base for the Rails instrumentation, nothing seems really dependent on the latest versions of Rails, yet there is a hard requirement to be on Rails 5.2 or above. Is there a reason why 5.2 is the minimal version other than ease of support? I want to make sure I'm not missing a key dependency or something like that.

**Comment by arielvalentin** [comment]
The "Library Compatibility" section in the [README](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/README.md) provides our reasoning for this. 

We don't guarantee or provide support for EoL versions of Ruby or any gems. 

Rails 5.2 and Ruby 2.7 are EoL and no longer receiving security updates. You will have to install and pin earlier versions of the instrumentation libraries if you want support for those versions. 

We will not provide any support or feature requests for any EoL versions should you run into a problem.
